SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -135,6 +135,10 @@
 
   private long zookeeperAsyncTimeout = 5000;
 
+  private int coreThreadpoolSize = 8;
```

Comment:
I was thinking we should have it be a Provider like a Factory that creates a new Managed Executor Service. That way we don't have to hard code the # of threads, and it also guarantees that each user can not mistakenly over-use threads in the pool and starve another process.